release/release_gold.sh

```diff
@@ -48,7 +50,25 @@ function check_usage {
 	LIFERAY_COMMON_LOG_DIR="${_PROMOTION_DIR%/*}"
 }
 
+function commit_to_branch {
+	git add "${1}"
+
+	git commit -m "${2}"
+
+	git push upstream "${3}"
+
+	if [ "${?}" -ne 0 ]
+	then
+		return "${LIFERAY_COMMON_EXIT_CODE_BAD}"
+	fi
+}
+
```

Comment:
Let's change to send a pull request to Brian instead, at least for now, to prevent conflicts between the release branch from the repos `liferay-portal-ee` and `brianchandotcom`.